apps/internal/base/internal/storage/storage.go

```diff
@@ -70,8 +70,10 @@ func checkAlias(alias string, aliases []string) bool {
 
 func isMatchingScopes(scopesOne []string, scopesTwo string) bool {
 	newScopesTwo := strings.Split(scopesTwo, scopeSeparator)
+	toLower(newScopesTwo)
 	scopeCounter := 0
 	for _, scope := range scopesOne {
+		scope = strings.ToLower(scope)
```

Comment:
Instead of doing this conversion, take a look at [`strings.EqualFold`](https://pkg.go.dev/strings#EqualFold).

Reply:
This is exactly what I wanted, making the change, thanks @jhendrixMSFT